conf/broker.conf

```diff
@@ -1281,8 +1281,8 @@ defaultNamespaceBundleSplitAlgorithm=range_equally_divide
 # load shedding strategy, support OverloadShedder and ThresholdShedder, default is ThresholdShedder since 2.10.0
 loadBalancerLoadSheddingStrategy=org.apache.pulsar.broker.loadbalance.impl.ThresholdShedder
 
-# load balance placement strategy, support LeastLongTermMessageRate and LeastResourceUsageWithWeight
-loadBalancerLoadPlacementStrategy=org.apache.pulsar.broker.loadbalance.impl.LeastLongTermMessageRate
+# load balance placement strategy, support LeastLongTermMessageRate and LeastLongTermMessageRate
```

Comment:
This line needs to keep the same as the original

Reply:
yes.

Reply:
OK , I will fix